src/driver/AsyncJobs.ts

```diff
@@ -1,19 +1,36 @@
 export interface IAsyncJobs {
     pushJob(job: Promise<any>): void
+    pushJobs(jobs: Array<Promise<any>>): void
     jobsComplete(): Promise<void>
 }
 
 export class AsyncJobs implements IAsyncJobs {
     private jobs: Array<Promise<any>> = []
 
+    pushJobs(jobs: Array<Promise<any>>): void {
+        this.jobs.concat(jobs)
+    }
+
     pushJob(job: Promise<any>): void {
         this.jobs.push(job)
     }
 
     async jobsComplete(): Promise<void> {
-        // More jobs might be added by each job, so we can't just await Promise.all() here.
```

Comment:
Why did you remove the comment? I think it's quite helpful to explain that here.